proto/api/v1beta1/event.proto

```diff
@@ -41,6 +41,14 @@ message EventWrapper {
     Encryption encryption = 10;
     Compression compression = 11;
 
+    // Deduplication pointer; if this event wrapper is a duplicate then it should be
+    // omitted from standard queries depending on the deduplication strategy of the
+    // topic. The DuplicateID points to the event that contains the data referred to by
+    // this duplicated event wrapper. A duplicate event wrapper will likely have all of
+    // the metadata but no event data.
+    bool is_duplicate = 12;
+    bytes duplicate_id = 13;
```

Comment:
Sounds like we will need to rebuild the protocol buffers for the SDKs?

Reply:
Yes, but not urgent since we're adding protobuf fields -- so it should be backwards compatible.